backend/config/database.js

```diff
@@ -1,68 +1,46 @@
 // NOTE: The fallback names of the env variables are here for backward compatibility only.
 // Prefer using the explicit READ_ONLY_*/WRITE_ONLY_* configuration options.
+const NEAR_DATABASE_CONFIGS = JSON.parse(process.env.NEAR_DATABASE_CONFIGS);
+
+/*
+type NearDatabaseConfig = {
+  host: string;
+  database: string;
+  username: string;
+  password?: string;
+}
+
+type NearDatabaseConfigs = {
+  indexer: NearDatabaseConfig;
+  analytics: NearDatabaseConfig;
+  telemetry: NearDatabaseConfig;
+  writeTelemetry?: NearDatabaseConfig;
+}
+*/
+
 module.exports = {
   readOnlyIndexerDatabase: {
     dialect: "postgres",
-    host:
-      process.env.NEAR_READ_ONLY_INDEXER_DATABASE_HOST ||
-      process.env.NEAR_INDEXER_DATABASE_HOST,
-    database:
-      process.env.NEAR_READ_ONLY_INDEXER_DATABASE_NAME ||
-      process.env.NEAR_INDEXER_DATABASE_NAME,
-    username:
-      process.env.NEAR_READ_ONLY_INDEXER_DATABASE_USERNAME ||
-      process.env.NEAR_INDEXER_DATABASE_USERNAME,
-    password:
-      process.env.NEAR_READ_ONLY_INDEXER_DATABASE_PASSWORD ||
-      process.env.NEAR_INDEXER_DATABASE_PASSWORD,
+    password: process.env.NEAR_INDEXER_DATABASE_PASSWORD,
+    ...NEAR_DATABASE_CONFIGS.indexer,
```

Comment:
A bunch of variables were basically unreadable. I agree the situation is odd and a proper way to handle this would be interpolation, but it's not supported yet on `render`.

I had no idea there is such Kurtosis, but it seems that updating [its code](https://github.com/kurtosis-tech/near-kurtosis-module/blob/3679eb2acbe352cd1962985ce23b9d6cc55e0121/kurtosis-module/impl/services/explorer_backend.ts) wouldn't be much of a hassle.

We can add backward-compatibility by getting back all the variables eventually overwritten by `NEAR_DATABASE_CONFIGS`.